plugins/k8saudit-gke/pkg/k8sauditgke/k8sauditgke.go

```diff
@@ -34,7 +34,7 @@ const (
 	PluginName               = "k8saudit-gke"
 	PluginDescription        = "Read Kubernetes Audit Events for GKE from a Pub/Sub subscription"
 	PluginContact            = "github.com/falcosecurity/plugins"
-	PluginVersion            = "0.1.1"
+	PluginVersion            = "0.1.2"
```

Comment:
```suggestion
	PluginVersion            = "0.2.0"
```
Although the change introduced is small, it's still an addition (not a fix), so we should bump the minor version number.